### ISSUE TITLE
 Error while loading package - Something went wrong while fetching the package.     

### ISSUE THREAD

**Issue by MichaelQA1** [a85/POSTMan-Chrome-Extension#13250]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Sometimes, Postman encounters an issue loading packages. Manually clearing the cache and reloading resolves this, but since I'm running collections via GitHub Actions, this isn’t feasible to handle in code.

Error loading package '@martian-moon-978360/pre_request': "An error occurred while fetching the package."

### Steps To Reproduce

Run the collections via  GitHub Actions frequently or hourly, in parallel use the UI postman for testing

### Operating System

macOS

### Postman Version

Version 11.14.1 UI Version: 11.14.1-ui-240925-1856 Desktop Platform Version: 11.14.0 (11.14.0)

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by giridharvc7** [comment]
Are you using the Postman CLI to run the collections that use package library?

**Comment by MichaelQA1** [comment]
The Postman CLI setup works with collections linked to jobs using collection IDs (e.g., "Sanity_12go(dev_and_integration)" with "32090846-9170d568-d19a-4caf-a38b-e59053eff818"). While the system generally performs as expected, hourly runs occasionally encounter issues with package loading

**Comment by tracymazelin** [comment]
> Are you using the Postman CLI to run the collections that use package library?

Yes.  We are calling a package in a pre-request script in multiple tests.  The package successfully loads in some tests and not others.  We receive `Error: Cannot find package `@##/test-helpers` regularly.  It can be reproduced in the postman client and also when running via CI/CD.

**Comment by tom-mrnk-github** [comment]
Same thing here constantly occuring, I cannot see a pattern, as test runs sometimes fail, sometimes work, locally or in scheduled monitors as well.
``Post-response: ReferenceError: <package library> is not defined``

Starting by 10th of June...
